pyproject.toml

```diff
@@ -1,3 +1,70 @@
+[build-system]
+requires = [
+    "poetry-core==1.4.0",
+    "poetry-version-plugin==0.1.3"
+]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "kapitan"
+description = "Generic templated configuration management for Kubernetes, Terraform and other things"
+authors = [
+    "Ricardo Amaro <ramaro@kapicorp.com>"
+]
+documentation  = "https://kapitan.dev/"
+repository = "https://github.com/kapicorp/kapitan"
+readme = "README.md"
+keywords = ["jsonnet", "kubernetes", "reclass", "jinja"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Topic :: Software Development :: Build Tools",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+]
+license = "Apache-2.0"
+version = "0.31.0rc0"
```

Comment:
Just one fix: version should now be `0.31.0`

Reply:
Alright, fixed. I would also like to use something like https://github.com/tiangolo/poetry-version-plugin in next step.

Reply:
Nice one, didn't know about it! That's a great idea :D

Reply:
I've used similar solution in my personal projects, but there I've used https://flit.pypa.io/en/stable/

https://smarie.github.io/python-getversion/